spec/javascripts/health-care/components/social-security-number.spec.jsx

```diff
@@ -1,75 +1,63 @@
 import React from 'react';
-import ReactTestUtils from 'react-addons-test-utils';
+import SkinDeep from 'skin-deep';
+
 import SocialSecurityNumber from '../../../../_health-care/_js/_components/social-security-number';
 
 describe('<SocialSecurityNumber>', () => {
-  let component = null;
-
-  beforeEach(() => {
-    component = ReactTestUtils.renderIntoDocument(
-      <SocialSecurityNumber ssn={{ ssn: '999-99-9999' }} onUserInput={(_update) => {}}/>
-    );
-    assert.ok(component, 'Cannot even render component');
+  describe('propTypes', () => {
+    let consoleStub;
+    beforeEach(() => {
+      consoleStub = sinon.stub(console, 'error');
+    });
+
+    afterEach(() => {
+      consoleStub.restore();
+    });
+
+    it('ssn is required', () => {
+      SkinDeep.shallowRender(<SocialSecurityNumber/>);
```

Comment:
...yeah....uh...so, apparently errors are printed for both the current component as well as the nested components. So consoleStub was being called twice. This check was also overkill anyways. I probably should remove from the other test since it's over-specified then.